gdx/src/com/badlogic/gdx/graphics/glutils/GLFrameBuffer.java

```diff
@@ -174,14 +182,33 @@ private void build () {
 
 		int result = gl.glCheckFramebufferStatus(GL20.GL_FRAMEBUFFER);
 
+		if (result == GL20.GL_FRAMEBUFFER_UNSUPPORTED && hasDepth && hasStencil
+			&& gl.glGetString(GL20.GL_EXTENSIONS).contains("GL_OES_packed_depth_stencil")) {
+			if (hasDepth) gl.glDeleteRenderbuffer(depthbufferHandle);
+			if (hasStencil) gl.glDeleteRenderbuffer(stencilbufferHandle);
```

Comment:
Can you reset the handles to zero? And add some javadocs to the `getDepthBufferHandle` and `getStencilBufferHandle` methods that they might return zero even when created with depth/stencil buffer.